scenarios/.env.stc-leafspine

```diff
@@ -34,3 +34,4 @@ P4RT_UP4_METERS=2
 P4RT_UP4_FLOWS_APP_FILTER=2
 # Used in pfcp-* scenarios to check UP4 flows, should be ENODEBS * 5
 PFCP_UP4_FLOWS=5
+PFCP_UP4_APP_FILTER_FLOWS=15
```

Comment:
Please, add a comment on how to calculate this value (see above).
Also, we miss this value for the singlepair setup